openassessment/templates/openassessmentblock/grade/oa_grade_waiting.html

```diff
@@ -16,13 +16,7 @@ <h2 class="step__title">
         <div class="wrapper--step__content">
             <div class="step__content">
                 <div class="grade__value__description">
-                    {% if waiting == 'peer' %}
-                        <p>{% trans "Your response is still undergoing peer assessment. After your peers have assessed your response, you'll see their comments and receive your final grade." %}</p>
-                    {% elif waiting == 'example-based' %}
-                        <p>{% trans "Your response is still undergoing example-based assessment. After your response has been assessed, you'll see the comments and receive your final grade." %}</p>
-                    {% elif waiting == 'all' %}
-                        <p>{% trans "Your response is still undergoing peer assessment and example-based assessment. After your peers have assessed your response and example-based assessment is complete, you'll see your peers' comments and receive your final grade." %}</p>
-                    {% endif %}
+                    <p>{% trans "Not all assessment steps have been completed for your response. After all assessment steps are complete, you will see feedback from everyone who assessed your response, and you will receive your final grade." %}</p>
```

Comment:
@catong @explorerleslie @cahrens I copied the already-discussed waiting string here as well, should this be different based on context?

Screenshot, this particular line is defining the top instance of the string shown, the already-discussed change is defining the bottom one:
<img width="867" alt="screen shot 2016-01-05 at 5 37 52 pm" src="https://cloud.githubusercontent.com/assets/7373924/12129687/304ba174-b3d3-11e5-9163-a156aad5ab2e.png">